site/CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```

Comment:
Could this be simply a symlink to https://github.com/kubernetes-sigs/kueue/blob/main/CONTRIBUTING.md?

If this is not part of the site, I would just remove it, given that there is already a file in the root directory.

Reply:
If people are going to the docsite to learn about how to contribute I think its useful to have it here.

Reply:
If anything, we should link the repo Contributing guide to point to docsite. e.g- kubeflow, knative, kind etc

Reply:
did you mean docsy?

This file seems confusing because it's talking about Google CLA, which is not relevant to this repo.